### PR TITLE
Multiple improvements to image service code and tests

### PR DESCRIPTION
**Pull request by paskal**
These are changes from #992 which are not relevant to the purpose of that PR, cut into separate PR to simplify 992 code review.

- rename variables according to golangci-lint recommendation
- make consistent returns in bolt_store
- less magic consonants in tests
- make commitTTL equal to EditDuration (opposed to 1.5 previously), so that image is committed to permanent storage after the comment can no longer be edited.
- move cleanupTTL to the Cleanup function, as it's not used elsewhere in the code.

I advise merging this with rebase.

### CODE REVIEW

File: backend/app/store/image/image.go [hunk 1]
```diff
@@ -186,17 +175,18 @@ func (s *Service) ExtractPictures(commentHTML string) (ids []string, err error)
 	return ids, nil
 }
 
-// Cleanup runs periodic cleanup with cleanupTTL. Blocking loop, should be called inside of goroutine by consumer
+// Cleanup runs periodic cleanup with 2.5*ServiceParams.EditDuration. Blocking loop, should be called inside of goroutine by consumer
 func (s *Service) Cleanup(ctx context.Context) {
-	log.Printf("[INFO] start pictures cleanup, staging ttl=%v", s.cleanupTTL)
+	cleanupTTL := s.EditDuration * 25 / 10 // cleanup images older than 1.5 * EditDuration
```
Comment (umputun) [inline backend/app/store/image/image.go:180]: well, 25/10 is 2.5 not 1.5. Maybe just do direct *1.5 (or *2.5)?

**Review by umputun — CHANGES_REQUESTED** [review]
looks ok, except the cleanupTTL confusion

File: backend/app/store/image/image.go (hunk 1 above)
Comment (paskal) [reply backend/app/store/image/image.go]: Fixed. There is type confusion otherwise, that is a way to keep numbers straight without types conversions.

**Review by umputun — APPROVED** [review]
LGTM

**Pull request merged**